functions/src/config/settings.ts

```diff
@@ -24,8 +24,9 @@ export const settings = {
   },
   dockerhub: {
     host: 'https://index.docker.io/v1',
-    baseRepository: 'unityci/base',
-    hubRepository: 'unityci/hub',
-    editorRepository: 'unityci/editor',
+    repositoryBaseName: 'unityci',
+    baseImageName: 'base',
+    hubImageName: 'hub',
+    editorImageName: 'editor',
```

Comment:
I see `repositoryBaseName` was added. Are we sure removing `unityci/` prefix here is fine?

Reply:
It's fine indeed. The settings are only used from the Dockerhub service class, which was also refactored, to handle repository name and image name separately.